src/DependencyInjection/LeagueOAuth2ServerExtension.php

```diff
@@ -280,10 +296,20 @@ private function configureDoctrinePersistence(ContainerBuilder $container, array
 
         $container->setParameter('league.oauth2_server.persistence.doctrine.enabled', true);
         $container->setParameter('league.oauth2_server.persistence.doctrine.manager', $entityManagerName);
+
+        if ($disableAccessTokenSaving) {
+            $container->setParameter('league.oauth2_server.persistence.doctrine.access_token.disabled', true);
+        } else {
+            $container->setParameter('league.oauth2_server.persistence.doctrine.access_token.enabled', true);
+        }
```

Comment:
yes it's mandatory for configureDoctrineExtension. I need both because I didn't find how to invert boolean value from string parameter. But maybe with php approach I could remove it